modules/ROOT/pages/try-scope-concept.adoc

```diff
@@ -97,14 +100,15 @@ Suppose now we add an Error Handler at flow level:
 </flow>
 ----
 
-We analyze this new scenario:
+The behavior in this scenario is:
+
+* Ignore (`INDIFFERENT`): The transaction is _continued_. This time, the commit of the `jms:publish` message depends on the flow. Because the error is now handled by an `on-error-continue` error handler, the transaction is committed. The message read from the `jms:listener` source is consumed, and the message processed by the `jms:publish` is actually sent.
+* Always Begin (`ALWAYS_BEGIN`): This raises an error because an active transaction already exists.
+* Begin or Join (`BEGIN_OR_JOIN`): Same behavior as `INDIFFERENT`.
 
-* Ignore (`INDIFFERENT`): Once again, the transaction is _suspended_, and the message is sent in the `jms:publish` operation regardless of the error caused by the `raise-error` processor. Then, as the error is handled in the `on-error-continue` Error Handler, the transaction is committed and the message is consumed.
-* Always Begin (`ALWAYS_BEGIN`): Since the transaction that runs within the Try Scope is a different one from the flow, this one is rolled back (since it is not handled) and the error propagated and handled at flow level. Therefore, the message will still be consumed, but the message sent with the `jms:publish` operation not (that transaction is rolled back)
-* Begin or Join (`BEGIN_OR_JOIN`): As mentioned in the previous example, the commit of the `jms:publish` message depends on the flow. Since the error is now handled by an `on-error-continue` Error Handler, the transaction is committed. Thus, both the message read from the `jms:listener` source is consumed and the message sent with `jms:publish` is actually sent.
+==== Error Handler Inside the Try Scope
 
-Suppose now, instead of adding an Error Handler at flow level, we add it in the `try` Scope, and the error is thrown
-after the execution of the scope:
+In this case, we add the error handler in the `try` Scope, and the error is thrown after the execution of the scope:
```

Comment:
"In this case, we add the error handler in the `try` Scope, and the error is thrown...."
  >  "In this case, adding the error handler in the `try` Scope causes the error to occur...."